### DISCUSSION TITLE
여쭤보고 싶은게 있습니다

### DISCUSSION THREAD

**Discussion by lazyflog** [QuadFlask/react-native-naver-map#88] [Q&A] [answered]
혹시 showsMyLocationButton의 커스텀에 대한게 있나요? (크기, 위치, 이미지)
아무리찾아봐도 안보여서 질문드립니다

**Comment by QuadFlask** [comment, marked as answer]
https://navermaps.github.io/android-map-sdk/guide-ko/4-1.html
위 페이지에서 지원하지 않는 기능은 제공하지 않습니다.
